implementations/rust/ockam/ockam_command/tests/bats/orchestrator_enroll/nodes.bats

```diff
@@ -201,3 +201,15 @@ EOF
   # Use the identity to send a message
   $OCKAM message send hi --identity i1 --to "/project/default/service/forward_to_$RELAY_NAME/secure/api/service/echo"
 }
+
+@test "nodes - create with config, using the specified enrollment ticket" {
```

Comment:
Maybe it would be good to add a bats test showing that we can override values coming from the configuration file with command-line arguments.